scripts/single_testsuite.py

```diff
@@ -102,6 +103,25 @@ def parse_directories(args):
                 return root_directory, install_directory
     return None, None
 
+def parse_target_test(target_test: str):
+    """Returns string dejagnu expects"""
+    test_case = os.path.basename(target_test)
+    dir_path = "./"
+    if not os.path.isdir(os.path.join(dir_path, "gcc")):
+        dir_path = "../"
```

Comment:
Why do we check if GCC is inside the scripts folder before checking the `../gcc` dir?

Reply:
Wasn't sure where this was going to be ran. If ran in build/ scripts/ directory then need `../gcc` but running it in the tree root would just be `./gcc`

Reply:
I think it's safe to assume it will always run in scripts, but I don't think it'll be a ton of harm to leave it in.